airbyte-integrations/connectors/source-metabase/source_metabase/manifest.yaml

```diff
@@ -16,7 +16,6 @@ definitions:
       type: "SessionTokenAuthenticator"
       username: "{{ config['username'] }}"
       password: "{{ config['password'] }}"
-      api_url: "{{ config['instance_api_url'] }}"
```

Comment:
remove the field from the manifest because it's now injected from the requester's url_base